src/sentry/static/sentry/app/components/events/interfaces/stacktraceLink.tsx

```diff
@@ -97,6 +139,7 @@ class StacktraceLink extends AsyncComponent<Props, State> {
       showModal: false,
       sourceCodeInput: '',
       match: {integrations: []},
+      isDismissed: null,
```

Comment:
@MeredithAnya Are we trying to differentiate between `null` and `false`? That could potentially lead to bugs if we aren't careful.